lib/tasks/test_vmdb.rake

```diff
@@ -3,7 +3,7 @@ require_relative "./evm_test_helper"
 if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
   namespace :vmdb do
-    task :setup => :setup_db
+    task :setup => [:verify_no_db_access_loading_rails_environment, :setup_db]
```

Comment:
> Yeah, I'm not sure about this. May be simpler to run it in a subprocess?

It will only be simpler if the followup tasks don't react too kindly to rails environment being loaded already.

Let's see what happens on travis and revisit.